cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_writer.py

```diff
@@ -0,0 +1,52 @@
+'''
+Created on Mar 4, 2025
+
+@author: pleyte
+'''
+from json import JSONEncoder
+import json
+
+from edu.ohsu.compbio.txeff.variant_transcript import VariantTranscript
+
+class VariantTranscriptEncoder(JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, VariantTranscript):
+            return {
+                "id": obj._id,
+                "chromosome": obj.chromosome, 
+                "position": obj.position, 
+                "reference": obj.reference, 
+                "alt": obj.alt,
+                "variantEffect": obj.variant_effect,
+                "variantType": obj.variant_type,
+                "aminoAcidPosition": obj.amino_acid_position,
+                "basePosition": obj.base_position,
+                "exon": obj.exon,
+                "gene": obj.gene,
+                "cDot": obj.c_dot,
+                "pDot1": obj.p_dot1,
+                "pDot3": obj.p_dot3,
+                "splicing": obj.splicing,
+                "cdnaTranscript": obj.cdna_transcript,
+                "proteinTranscript": obj.protein_transcript,
+                "sequenceVariant": obj.sequence_variant,
+                "referenceContext": obj.reference_context
+            }
+        else:
+            return super().default(obj)
+        
+class TxEffWriter(object):
+    '''
+    Writes variant transcripts to file 
+    '''
+    def __init__(self, out_file):
+        '''
+        Constructor
+        '''
+        self._out_file = out_file
+    
+    def write(self, variant_transcripts: list):
+        '''
+        Write transcripts to output file 
+        '''
+        json.dump(sorted(variant_transcripts), self._out_file, cls=VariantTranscriptEncoder, indent=2)
```

Comment:
I need to update this code so the json being created matches [src/main/java/edu/ohsu/cgd/api/response/TranscriptEffects.java](https://source.ohsu.edu/kdls/cgd/pull/163/files#diff-88ea6f93f96e8ee130b1a03714c3d3d346ebfed79e0f32302cac4f2e09288806)
* Right now it is just writing out a list of transcript effects.  
* Need to add tfxVersion